src/components/SimpleTable.jsx

```diff
@@ -83,6 +83,9 @@ const DataRows = ({rowData, baseStyle, columnHeaders, rowWrapper = ({renderedRow
   return rowData.map((row, index) => {
     const id = rowData[index][0].id;
     const mapKey = id || `noId-index-${index}`;
+    if (rowData[index][0].label === "display-names") {
+      baseStyle.backgroundColor = index % 2 === 0 ? 'white' : '#e2e8f4';
+    }
```

Comment:
The logic here is a little confusing - it's not clear to me why `label === "display-names"` should indicate a striped table condition. I think a more legible approach would be to add an optional condition (perhaps `striped`) to the row object that indicated a striped condition. Alternatively, there could be a custom style passed in on the row object that would be merged in on line 90.